x-pack/plugins/search_playground/server/lib/conversational_chain.test.ts

```diff
@@ -109,15 +109,14 @@ describe('conversational chain', () => {
 
       const streamToValue: string[] = await new Promise((resolve, reject) => {
         const reader = stream.getReader();
```

Comment:
now no longer using a bytesOutput for langchain and therefore don't need to decode it.